x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/transport/netty4/SecurityNetty4Transport.java

```diff
@@ -186,11 +191,14 @@ protected void initChannel(Channel ch) throws Exception {
         private final boolean hostnameVerificationEnabled;
         private final SSLConfiguration sslConfiguration;
         private final SSLService sslService;
+        private final String sniServerName;
```

Comment:
This should maybe be a list?

Reply:
++

Reply:
Should it be a list? We only support one value for an attribute.